packages/state-manager/src/sagas/files/checkForMissingFiles/checkForMissingFiles.saga.test.ts

```diff
@@ -171,17 +171,143 @@ describe('checkForMissingFilesSaga', () => {
       .run()
   })
 
+  test('download only specific file from many missing files', async () => {
+    const initialState = prepareStore().store
+
+    const factory = await getFactory(initialState)
+
+    const community = await factory.create<
+      ReturnType<typeof communities.actions.addNewCommunity>['payload']
+    >('Community')
+
+    const alice = await factory.create<
+      ReturnType<typeof identity.actions.addNewIdentity>['payload']
+    >('Identity', { id: community.id, nickname: 'alice' })
+
+    const message1 = Math.random().toString(36).substr(2.9)
+    const message2 = Math.random().toString(36).substr(2.9)
+
+    const channelAddress = 'general'
+
+    const missingFile1: FileMetadata = {
```

Comment:
I think it might be good to name those files by what they really are, eg. "missingFileCanceled" and "missingFilePending". It'll make the test easier to read